ipc-addr.c

```diff
@@ -18,3 +18,8 @@ bool check_env(void) {
 
 	return true;
 }
+
+int get_ipc_address(char *address, size_t size) {
```

Comment:
Maybe the `check_env` checks be moved to this function? We wouldn't rely on `check_env` being called prior to this one. Calling `get_ipc_address` without `check_env` results in a wrong address at best, or undefined behavior in the worst case (`printf` with NULL).

Reply:
Sounds like a nice idea.

Reply:
Hm, I did this, but it would mean we don't check `WAYLAND_DISPLAY` when IPC isn't built.

Reply:
Right. This is probably not that of a big deal.